content/en/hypervisor.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Hypervisor
+status: Feedback Appreciated
+category: Technology
+tags: ["application", "", ""]
+---
+
+## What it is
+
+A hypervisor is a software application that performs [virtualization](/virtualization/)
```

Comment:
@KiranSatyaRaj 
I think `software application` may not correct technically.
In general, there are two types of hypervisor. 
 1) bare metal/native hypervisor that runs on bare metal computer directly (e.g., KVM, Xen, ...)
 2) hosted hypervisor that runs on OS as an application (e.g., Virtual box)

So, if we say `A hypervisor is a software application`, readers may be confused with the second type of hypervisor.